### PR TITLE
Restrict allowed TLS versions.

### PR DESCRIPTION
**Pull request by udhos**

### CODE REVIEW

File: wuzz.go [hunk 1]
```diff
@@ -1188,6 +1188,31 @@ func (a *App) ParseArgs(g *gocui.Gui, args []string) error {
 			}
 		case "-k", "--insecure":
 			a.config.General.Insecure = true
+		case "-T", "--tls":
+			if arg_index >= args_len-1 {
+				g.Close()
+				log.Fatal("Missing TLS version range: MIN,MAX")
```
Comment (asciimoo) [inline wuzz.go]: Please rewrite to return with an error to break argument parsing instead of the above two lines.

File: wuzz.go [hunk 2]
```diff
@@ -1394,3 +1426,10 @@ func main() {
 		log.Panicln(err)
 	}
 }
+
+var tlsVersions = map[string]uint16{
+	"SSL3.0": tls.VersionSSL30,
+	"TLS1.0": tls.VersionTLS10,
+	"TLS1.1": tls.VersionTLS11,
+	"TLS1.2": tls.VersionTLS12,
+}
```
Comment (asciimoo) [inline wuzz.go]: Global variables are upper case and defined in the top of the file, please follow these conventions.

**Comment by udhos** [comment]
Requested changes have been added.

**Comment by asciimoo** [comment]
Great, works well. One more thing: could you rebase this PR to the latest master?
Thank you

**Comment by udhos** [comment]
Rebased.

**Comment by asciimoo** [comment]
@udhos awesome, thank you.

**Pull request merged**